src/OpenTracing/NoopSpan.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace OpenTracing;
+
+use Nanotime\Nanotime;
+
+class NoopSpan implements Span
+{
+    public static function create()
+    {
+        return new self();
+    }
+
+    public function operationName()
+    {
+        return 'noop_span';
+    }
+
+    public function context()
+    {
+        return null;
+    }
+
+    public function finish(Nanotime $finishTime = null, $logRecords = [])
+    {
+    }
+
+    public function overwriteOperationName($newOperationName)
```

Comment:
I'd avoid the `set` as they are associated to anti-patterns, in addition the domain language from specification describes the use car `overwrite operation name` so I'd keep like that: https://github.com/opentracing/specification/blob/master/specification.md#overwrite-the-operation-name